src/components/presentationals/ArticleLists/ArticleLists.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ArticleLists from './ArticleLists';
+
+const articles = [
+  {
+    id: 123456789,
+  },
+];
+const emptyArticles = [];
+
+describe('ArticleLists', () => {
+  it('should render correctly', () => {
+    const component = mount(<ArticleLists articles={articles} />);
+
```

Comment:
We can remove this line space since this is just a two line test.

Reply:
This is has been removed